### PR TITLE
fix(tests): handle edge case in probabilistic tests

### PR DESCRIPTION
**Pull request by bmzig**
In the tryMulticall tests, since we randomly determine if a transaction succeeds or fails, there is a chance that all the transactions in a bundle fail, meaning that we discard the bundle entirely and do not execute any transaction. Our checks assume that this does not happen, so this PR adds the edge case in our tests so we will no longer fail on these tests on occasion.

### CODE REVIEW

File: test/TryMulticallClient.ts [hunk 1]
```diff
@@ -417,10 +421,21 @@ describe("TryMulticallClient", async function () {
             mrkdwn: `This transaction is expected to ${succeed === 0 ? "fail" : "pass"} simulation.`,
           } as AugmentedTransaction;
           multiCaller.enqueueTransaction(txn);
+          queueCopy.push(succeed);
+        }
+        // Determine if all 5 transactions in a bundle failed. This assumes chunkSize | nTxns.
+        let emptyBundles = 0;
+        const nBundles = nTxns / chunkSize;
```
Comment (pxrl) [inline test/TryMulticallClient.ts:428]: Does it make sense to round this to an integer?

Comment (bmzig) [reply test/TryMulticallClient.ts:428]: It assumes that `nTxns` is a multiple of `chunkSize`, so there should theoretically be no rounding here.

**Review by pxrl — APPROVED** [review]
